source/material_model/visco_plastic.cc

```diff
@@ -1072,38 +1072,31 @@ namespace aspect
+                                   "Jaumann corotational stress rate. This later term (eqn. 24) contains the "
+                                   "time derivative of the deviatoric stress ($\\dot{\\tau}$) and terms that "
+                                   "account for material spin (e.g., rotation) due to advection: "
+                                   "$\\overset{\\nabla}{\\tau} = \\dot{\\tau} + {\\tau}W -W\\tau$. "
                                    "Above, $W$ is the material spin tensor (eqn. 25): "
                                    "$W_{ij} = \\frac{1}{2} \\left (\\frac{\\partial V_{i}}{\\partial x_{j}} - "
                                    "\\frac{\\partial V_{j}}{\\partial x_{i}} \\right )$. "
                                    "\n\n "
-                                   "The Jaumann stress-rate can also be approximated using terms from the time "
-                                   "at the previous time step ($t$) and current time step ($t + \\Delta t^{e}$): "
-                                   "$\\smash[t]{\\overset{\\triangledown}{\\tau}}^{t + \\Delta t^{e}} \\approx "
+                                   "The Jaumann stress-rate can also be approximated using terms from the "
+                                   "previous time step ($t$) and current time step ($t + \\Delta t^{e}$): "
+                                   "$\\smash[t]{\\overset{\\nabla}{\\tau}}^{t + \\Delta t^{e}} \\approx "
                                    "\\frac{\\tau^{t + \\Delta t^{e} - \\tau^{t}}}{\\Delta t^{e}} - "
                                    "W^{t}\\tau^{t} + \\tau^{t}W^{t}$. "
                                    "In this material model, the size of the time step above ($\\Delta t^{e}$) "
                                    "can be specified as the numerical time step size or an independent fixed time "
-                                   "step. If the latter case is a selected, the user has an option to apply a "
+                                   "step. If the latter case is selected, the user has an option to apply a "
                                    "stress averaging scheme to account for the differences between the numerical "
                                    "and fixed elastic time step (eqn. 32). If one selects to use a fixed elastic time "
                                    "step throughout the model run, this can still be achieved by using CFL and "
-                                   "maximum time step values that restrict the numerical time step to a specific time. "
+                                   "maximum time step values that restrict the numerical time step to a specific time."
                                    "\n\n "
-                                   "The formulation above allows rewriting the total rate of deformation (eqn. 29) as "
+                                   "The formulation above allows rewriting the total rate of deformation (eqn. 29) as\n "
```

Comment:
not actually necessary, but it's not going to hurt either.

Reply:
When printed in the manual this equation hangs off the line and the end of the equation can be read.